### ISSUE TITLE
[AdaptiveCards- Main Body Content]: Labels are not associated with the elements under “AdaptiveCards” main section.

### ISSUE THREAD

**Issue by carlos-zamora** [microsoft/AdaptiveCards#7275]
https://microsoft.visualstudio.com/OS/_workitems/edit/35894452

**Comment by vagpt** [comment]
This issue is not repro, hence closing this issue.
